### PR TITLE
add gaudi driver chart v0.1.1

### PR DESCRIPTION
**Pull request by oxxenix**

### CODE REVIEW

**Comment by oxxenix** [comment]
@byako, @eero-t, PTAL

File: charts/intel-gaudi-resource-driver/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Dynamic Resource Allocation (DRA) Intel Gaudi Driver Helm Chart
+
+## The chart installs Gaudi resource driver:
+
+- [Gaudi](https://github.com/intel-innersource/containers.orchestrators.kubernetes.intel-resource-drivers-for-kubernetes/doc/gaudi/README.md)
```
Comment (byako) [inline charts/intel-gaudi-resource-driver/README.md:5]: This is internal repository, nobody from outside Intel will be able to open this link. Same about other places, like two lines below. Needs to be replaced with public GH repository address.

Comment (byako) [reply charts/intel-gaudi-resource-driver/README.md:5]: https://github.com/intel/helm-charts/pull/58/files#r1708947878

File: charts/intel-gaudi-resource-driver/crds/gpu.resource.intel.com_gpuallocationstates.yaml
Comment (byako) [inline charts/intel-gaudi-resource-driver/crds/gpu.resource.intel.com_gpuallocationstates.yaml:1]: gpuallocationstates? Wrong file ?

File: charts/intel-gaudi-resource-driver/crds/gpu.resource.intel.com_gpuclaimparameters.yaml
Comment (byako) [inline charts/intel-gaudi-resource-driver/crds/gpu.resource.intel.com_gpuclaimparameters.yaml:1]: gpuclaimparameters? wrong file?

File: charts/intel-gaudi-resource-driver/crds/gpu.resource.intel.com_gpuclassparameters.yaml
Comment (byako) [inline charts/intel-gaudi-resource-driver/crds/gpu.resource.intel.com_gpuclassparameters.yaml:1]: gpuclassparameters? Wrong file?

File: charts/intel-gaudi-resource-driver/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{{/* Define common helpers for both GPU */}}
```
Comment (byako) [inline charts/intel-gaudi-resource-driver/templates/_helpers.tpl:1]: `for both GPU` ?

File: charts/intel-gaudi-resource-driver/templates/serviceaccount.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{{- if .Values.serviceAccount.create -}}
```
Comment (byako) [inline charts/intel-gaudi-resource-driver/templates/serviceaccount.yaml:1]: Will the resource driver work without service account? Why are we giving the choice to not install it ?

**Review by byako — CHANGES_REQUESTED** [review]

File: charts/intel-gaudi-resource-driver/README.md (hunk 1 above)
Comment (oxxenix) [reply charts/intel-gaudi-resource-driver/README.md]: yes, I will change the doc for gpu driver in a separate PR not to interfere with this one

**Review by byako — APPROVED** [review]

**Pull request merged**